Include/cpython/object.h

```diff
@@ -39,13 +39,17 @@ typedef struct _Py_Identifier {
     // Index in PyInterpreterState.unicode.ids.array. It is process-wide
     // unique and must be initialized to -1.
     Py_ssize_t index;
+    // Hidden PyMutex struct for non free-threaded build.
+    struct {
+        uint8_t v;
+    } mutex;
```

Comment:
This is a bit icky. I think it's okay here considering that we are not really using `_Py_Identifier` anywhere, but let's not repeat this pattern elsewhere.

Reply:
Agree :)